alpheios_nemo_ui/data/assets/scss/alpheios.scss

```diff
@@ -1,25 +1,25 @@
-@import url('https://fonts.googleapis.com/css?family=Amiri');
-@import url('https://fonts.googleapis.com/css?family=Source+Serif+Pro');
-@import url('https://fonts.googleapis.com/css?family=Source+Sans+Pro');
-
-@import "bootstrap.scss";
-@import "bootstrap_overrides";
-
-@import "common";
-
-@import "style-guide.scss";
-@import "design-commons";
-
-@import "help-popup";
-@import "bibliog-popup";
-@import "copyright-popup";
-
-@import "main-dropdown";
-
-@import "main-navbar";
-@import "breadcrumb-navbar";
-@import "navlogin";
-
-@import "collections";
-@import "references";
-@import "text";
+@import url('https://fonts.googleapis.com/css?family=Amiri');
+@import url('https://fonts.googleapis.com/css?family=Source+Serif+Pro');
+@import url('https://fonts.googleapis.com/css?family=Source+Sans+Pro');
+
+@import "bootstrap.scss";
+@import "bootstrap_overrides";
+
+@import "common";
+
+@import "style-guide";

```

Comment:
but it's now named _style-guide.css with a leading underscore... will the import work??

Reply:
yes, I have made it with underscore - similiar as in ruby.
Because in scss the following imports are all the same
```
@import "foo";
@import "foo.scss";
@import "_foo";
@import "_foo.scss";
```

It is a good practice to add _ to something that has only vars and mixins.
In ruby/middleman it is important to add _ because otherwise it inserts it many times.
But in flask it checks somehow and add it only once.

Anyway I have tested - everything is working.

Here it is a discussion about underscores in scss
https://stackoverflow.com/questions/27979693/sass-import-using-leading-underscore

Reply:
ok, thanks!